Modules/xASL_module_ASL.m

```diff
@@ -261,6 +261,8 @@
 %% G1. Split ASL and M0 within the ASL time series
 % Run this when the data hasn't been touched yet
 % The first three states are here, because the first two are run only conditionally
+
+% NOTE THAT THIS PART IS BEING PHASED OUT, AND NOW PROCESSED IN `xASL_bids_parseM0`
```

Comment:
I agree. But  we might write a bit more explaining that this is done in BIDS->Legacy conversion and we keep this here only when we need to work with data imported long time ago.

Reply:
Done